requirements.txt

```diff
@@ -1,7 +1,12 @@
 cwrap
-numpy
-pandas
-sphinx
+functools32;python_version=='2.7'
 future
+ninja
+numpy<=1.16.6
```

Comment:
Any particular reason for upper bounding the numpy and pandas version?

Reply:
Due to Python 2.7 -- Pandas 1.0 dropped support for it and the latest is 0.25.3 .

Had problems with latest numpy+py2, so I put an upper bound on the one that worked for me, but it's not very scientific.

Reply:
I don't mind an upper bound for Python 2.7, but then we should make it explicit for `python_version < "3"` and then have no upper bound for `python_version >= "3"

Reply:
Should be good now.